docs/bond/README.md

```diff
@@ -0,0 +1,98 @@
+# Bond interfaces with SR-IOV
+
+The below shows a method to create a pod with 2 SR-IOV interfaces bonded into a single bond interface inside a container using [Bond CNI](https://github.com/intel/bond-cni)
+
+This style of configuration can be used to create workloads with high availability - i.e. with two VFs from different PFs connected to the pod.
+
+## Prerequisites
+This guide assumes the quick set up of the repo has been followed. That is that Multus SRIOV CNI and SRIOV Device Plugin have been installed.
+
+Additionally at least two SRIOV Virtual Functions of the type 'intel_sriov-netdevice' should be available on a target node.
+
+
+## Installing Bond CNI
+In order to get Bond CNI working in your cluster first clone the repo:
+
+`git clone https://github.com/intel/bond-cni`
+
+Next move to the directory and build the binary:
+
+`cd bond-cni && ./build.sh`
+
+As with all CNI directories this binary will need to be placed in the CNI directory on each node in order to be used by the cluster. By default this directory is at `/opt/cni/bin`
+
+For a single node this can simply be copied:
+``cp bin/bond /opt/cni/bin``
+
+For each additional host in the cluster run:
+``scp bin/bond <USER>@<HOST_IP>:/opt/cni/bin``
+
+## Network Attachment Definitions
+Next create the network attachment definitions. The SRIOV CRD is similar to the one in the main repo - but it omits the IP address management as that will be handled by the bond interface. Note that this isn't obligatory but is standard when bonding interfaces.
+
+``kubectl apply -f sriov-net.yaml``
+
+Next create the bonded network attachment definition:
+
+``kubectl apply -f bond-crd.yaml``
+
+### Pod deployment
+With the above network definitions in place the following pod should deploy to a node with 2 SRIOV interfaces available and set up the SRIOV and Bond interfaces correctly.
+``
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+{"name": "sriov-net",
+"interface": "net1"
```

Comment:
Although Multus will name the interface with netx in order, but I think it's good to explicitly list the interface name here which maps to the bond-cni net-attach-def.